setup.py

```diff
@@ -19,7 +19,7 @@
 
 
 DEPENDENCIES = (
-    "cachetools>=2.0.0,<3.2",
+    "cachetools>=2.0.0,<5.0",
```

Comment:
v4 requires Python 3.5 or greater [cachetool's changelog](https://github.com/tkem/cachetools/blob/master/CHANGELOG.rst#v400-2019-12-15), [setup.cfg](https://github.com/tkem/cachetools/blob/fa9d4760dc44cd77f5eafd30cbaeed40a3435c96/setup.cfg#L27) via `python_requires`.

pip should install the last compatible version, so merging this shouldn't break any 2.7 users